test/client_test.coffee

```diff
@@ -15,14 +15,22 @@ describe 'Basic test for the 3Scale::Client', ->
       call = -> new Client()
       assert.throws call, 'missing provider_key'
 
-    it 'should have an default host', ->
+    it 'should have a default host', ->
       client = new Client(123)
       assert.equal client.host, 'su1.3scale.net'
 
     it 'can change the default host', ->
       client = new Client(123, 'example.com')
       assert.equal client.host, 'example.com'
 
+    it 'should have a default port', ->
+      client = new Client(123, 'example.com')
+      assert.equal client.port, '443'
```

Comment:
It is not an issue that the port is a string? I guess the underlying library converts it to number, right?